source/blueprints/screening_app.rst

```diff
@@ -0,0 +1,507 @@
+Health Screening
+================
+
+In this step-by-step walkthrough, you'll build a conversational application for a health screening that allows users to identify their risk of prediabetes.
```

Comment:
"...for a health screening that..." --> "...for a health screening application that.."